src/Git/RepoDetector.php

```diff
@@ -3,15 +3,14 @@
 
 class RepoDetector
 {
-    public function getDiffUrl(string $remote_url, string $from, $to): string
+    public function getDiffUrl(string $remoteUrl, string $from, $to): string
     {
-        if (
-            preg_match('#^git@github.com:(?P<org>[\w_-]+)/(?P<repo>[\w_-]+).git$#', $remote_url, $m)
-            || preg_match('#^https://github.com/(?P<org>[\w_-]+)/(?P<repo>[\w_-]+)(.git)?$#', $remote_url, $m)
-        ) {
+        $regExp = '#^(git@github.com:|https://github.com/)(?P<org>[\w_-]+)/(?P<repo>[\w_-]+).git$#';
```

Comment:
I am sorry for not writing the tests, but this will break the functionality as the `.git` suffis x is optional for the http case.